app/src/main/res/layout/feedinfo.xml

```diff
@@ -86,45 +86,57 @@
                     tools:background="@android:color/holo_green_dark"/>
 
             <TextView
-                    style="@style/AntennaPod.TextView.Heading"
-                    android:id="@+id/lblLanguage"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="8dp"
-                    android:layout_marginBottom="8dp"
-                    android:text="@string/language_label"
-                    android:textColor="?android:attr/textColorPrimary"
-                    tools:background="@android:color/holo_red_light"/>
+                android:id="@+id/lblUrl"
+                style="@style/AntennaPod.TextView.Heading"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="8dp"
+                android:layout_marginBottom="4dp"
+                android:text="@string/url_label"
+                android:textColor="?android:attr/textColorPrimary"
+                tools:background="@android:color/holo_red_light" />
 
             <TextView
-                    android:id="@+id/txtvLanguage"
+                android:id="@+id/txtvUrl"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="?attr/selectableItemBackground"
+                android:maxLines="4"
+                android:paddingTop="4dp"
+                android:paddingBottom="4dp"
+                tools:background="@android:color/holo_green_dark"
+                tools:text="http://www.example.com/feed" />
+
+            <TextView
+                android:id="@+id/lblStatistics"
+                style="@style/AntennaPod.TextView.Heading"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="8dp"
+                android:layout_marginBottom="8dp"
+                android:text="@string/statistics_label"
+                android:textColor="?android:attr/textColorPrimary"
+                tools:background="@android:color/holo_red_light" />
+
+            <TextView
+                    android:id="@+id/txtvStatistics"
```

Comment:
Could you please make this a `TableLayout`? Currently, based on the length of the labels, the statistics section looks a bit unaligned.